client/src/components/layout/RegisterForm.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Input, Form } from 'antd';
+import { UserOutlined } from '@ant-design/icons';
+import { REGISTER_ASYNC } from '../../sagas/types';
+
+const RegisterForm = ({ registerAsync }) => {
+  return (
+    <Form onFinish={formData => registerAsync(formData)}>
```

Comment:
@amichael497 do you know how to align all of the form items by left extent of the input field (rather than the left extent of the label), as well as left aligning the button with everything?

![image](https://user-images.githubusercontent.com/19275178/79271730-e9f27880-7e65-11ea-8871-d4a25554ea73.png)